src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -149,6 +133,7 @@ final class ExposureDetectionViewController: DynamicTableViewController, Require
 		case longGuide = "longGuideCell"
 		case link = "linkCell"
 		case hotline = "hotlineCell"
+		case survey = "surveyCell"
```

Comment:
Wondering why don't we keep these strings as constants in a separate file

Reply:
I think because they are only used in the context of this view controller 🤷‍♂️